ci-operator/config/openshift/openshift-tests-private/README.md

```diff
@@ -3,17 +3,36 @@ The intention of creating the Prow job naming regulation is to make sure we name
 # Job Configuration Location
 In https://github.com/openshift/release.git under the *ci-operator/config/openshift/openshift-tests-private* directory
 
-# File Naming
+## Configuration File Types
+There are two types of Prow configuration files under *openshift-tests-private*:
+- Infrastructure file
```

Comment:
@JianLi-RH I can only review this part. for others, I have no idea on them.
for this part,
```console
Pre-merge files: these files will be triggered when image file changes are submitted, it will generate new images or update existing images in image stream against the change in image file
```
should be 
```console
Pre-merge files: these files define the presubmit job which is triggered when the PR is created, so it is used for PR. It will generate new images based on PR's content and the images is used for presubmit job. the image is not promoted.
```

```console
Image files: these files define images,and export images into image stream.
```
to
```console
Image files: these files generate images based on PR's content and promote them so that the regression take this images.
```

Reply:
Thanks @kuiwang02